src/proto/semantifly.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+package semantifly;
+
+import "index.proto";
+
+option go_package = "accretional.com/semantifly/proto";
+
+service Semantifly {
+  rpc Add(AddRequest) returns (AddResponse) {}
+  rpc Delete(DeleteRequest) returns (DeleteResponse) {}
+  rpc Get(GetRequest) returns (GetResponse) {}
+  rpc Update(UpdateRequest) returns (UpdateResponse) {}
+  rpc LexicalSearch(LexicalSearchRequest) returns (LexicalSearchResponse) {}
+}
+
+message AddRequest {
+  repeated ContentMetadata files_data = 1;
+  bool make_copy = 2;
+}
+
+message AddResponse {
+  string error_message = 1;
+}
+
+message DeleteRequest {
+  bool delete_copy = 1;
+  repeated string data_uris = 2;
+}
+
+message DeleteResponse {
+  string error_message = 1;
+}
+
+message GetRequest {
+  string name = 1;
+}
+
+message GetResponse {
```

Comment:
Add ContentMetadata to the GetResponse and make content optional. It could be possible that the content is stored locally to the client/authenticated and not accessible by the server, in which case we'd want to tell the client how to access the data even if the server itself cannot

Reply:
fixed